src/settings.ts

```diff
@@ -254,23 +270,77 @@ export class ScaleSettings extends FormattingSettingsSimpleCard {
     public slices: FormattingSettingsSlice[] = [this.provideMinHeight, this.lnScale];
 }
 
-export class NodeComplexSettings extends FormattingSettingsSimpleCard {
+export class LinksOrder extends FormattingSettingsSimpleCard {
+    public name: string = "linkOrderGroup";
+    public displayNameKey: string = "Visual_LinksOrder";
+
+    public shouldReorder= new formattingSettings.ToggleSwitch({
+        name: "linksReorder",
+        displayNameKey: "Visual_AutoLinksReorder",
+        value: false,
+    });
+
+    slices: formattingSettings.Slice[] = [this.shouldReorder];
+}
+
+class PersistPropertiesGroup extends FormattingSettingsSimpleCard {
+    public name: string = "persistProperties";
+    public displayNameKey: string = "Visual_NodePositions";
+    public collapsible: boolean = false;
+    public visible: boolean = true;
     public nodePositions = new formattingSettings.ReadOnlyText({
         name: "nodePositions",
         displayNameKey: "Visual_NodePositions",
-        value: ""
+        value: "",
+        visible: false,
     }); 
 
     public viewportSize = new formattingSettings.ReadOnlyText({
         name: "viewportSize",
         displayNameKey: "Visual_ViewportSize",
-        value: ""
+        value: "",
+        visible: false,
+    });
+
+    public slices: FormattingSettingsSlice[] = [this.nodePositions, this.viewportSize]
+}
+
+export class ButtonSettings extends FormattingSettingsSimpleCard {
+    public static DefaultFill: string = "#DCDCDC";
```

Comment:
I advise to combine default properties into one object. e.g. `defaults = { fill: ...}` and creating an type interface for it.